{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/validation/notebooks/ModelValidation.py

```diff
@@ -65,6 +65,13 @@
 
 # COMMAND ----------
 
+{% if cookiecutter.include_feature_store == "yes" %}
+print(
+    "Currently model validation is not supported for models registered with feature store. Please refer to "
+    "issue https://github.com/databricks/mlops-stack/issues/70 for more details."
+)
+dbutils.notebook.exit(0){% endif %}
+
```

Comment:
On L117, we can remove `{% if cookiecutter.include_feature_store == "no" %}`. Also do we want to wrap the rest of the notebook in an else statement or we're ok with it still generating?

Reply:
I'm making the minimal changes so that if the issue https://github.com/databricks/mlops-stack/issues/70 is resolved, we can quickly reenable model validation.
Otherwise we can remove the validation notebook and skip the validation block in training-job.tf when feature store is used.

Reply:
The ignorePatterns in latest markdown-link-check is not working. 
It works when I specifies `markdown-link-check@3.10.3` in the test.

Reply:
Awesome let's get this merged ASAP so it unblocks the other PRs too